types/chart.js/index.d.ts

```diff
@@ -360,6 +361,8 @@ declare namespace Chart {
     }
 
     interface ChartTooltipModel {
+        body: object[];
```

Comment:
You forgot to read the comment in the documentation.

```
// Body
// The body lines that need to be rendered
// Each object contains 3 parameters
// before: string[] // lines of text before the line with the color square
// lines: string[], // lines of text to render as the main item with color square
// after: string[], // lines of text to render after the main lines
```
https://www.chartjs.org/docs/latest/configuration/tooltip.html

Therefore, I suggest you create an interface like this:
(Better name might be needed)

```typescript
interface BodyProperties {
  before: string[]
  lines: string[]
  after: string[]
}
```

And then use that interface:

```typescript
body: BodyProperties[]
```